docker/create_cluster.py

```diff
@@ -107,17 +128,19 @@ def create_cluster(clean, network_name, number_of_nodes, public_key_path):
 
 if __name__ == '__main__':
 
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--clean', help='Remove all user defined networks / container', action='store_true')
-    parser.add_argument('--network-name', help='Name of docker network', required=True)
-    parser.add_argument('--number-of-nodes', help='Number of nodes to create in the network', required=True)
-    parser.add_argument('--path-to-public-key', help='Number of nodes to create in the network', required=True)
-    args = parser.parse_args()
+    PARSER = argparse.ArgumentParser()
+    PARSER.add_argument('--pull', help='Specify whether to pull the latest docker containers', action='store_true')
```

Comment:
I would like to propose that this docker clean up not block this PR as I will be working on it right after in its own branch and there will be a lot of changes.